src/styles/blocks/header.scss

```diff
@@ -0,0 +1,52 @@
+.main {
```

Comment:
why do you put styles for main block in header.scss file? create a separate file for it